Entities/Common/Building/BlobPlacement.as

```diff
@@ -259,7 +259,7 @@ void onTick(CBlob@ this)
 						{
 							CBlob@ blob = b[nearblob_step];
 
-							if (blob is carryBlob || blob is this) continue;
+							if (blob is carryBlob || blob.hasTag("player")) continue;
```

Comment:
But then you still cannot build over some things like bushes, chickens or bisons, which should not block ladder building.
IMO it needs a broader check than the `player` tag.
I am not certain what, though.